tests/core/full_node/stores/test_full_node_store.py

```diff
@@ -47,7 +44,7 @@ def cleanup_keyring(keyring: TempKeyring):
 log = logging.getLogger(__name__)
 
 
-@pytest_asyncio.fixture(scope="function", params=[1, 2])
+@pytest_asyncio.fixture(scope="function", params=[1, 2])  # xxxtest_constants_original.replace
```

Comment:
```suggestion
@pytest_asyncio.fixture(scope="function", params=[1, 2])
```

Is this intentional?

Reply:
No, that's leftover